services/QuillLMS/app/services/diagnostic_insights/diagnostics_assigned_by_classroom.rb

```diff
@@ -4,20 +4,13 @@ module DiagnosticInsights
   class DiagnosticsAssignedByClassroom < ApplicationService
     attr_reader :classroom_id
 
-    STARTER_DIAGNOSTIC = :starter
-    INTERMEDIATE_DIAGNOSTIC = :intermediate
-    ADVANCED_DIAGNOSTIC = :advanced
-    ELL_STARTER_DIAGNOSTIC = :ell_starter
-    ELL_INTERMEDIATE_DIAGNOSTIC = :ell_intermediate
-    ELL_ADVANCED_DIAGNOSTIC = :ell_advanced
-
     DIAGNOSTIC_TYPE_MAPPING = {
```

Comment:
While this has the same name as the value that was moved to `Constants`, the version in this service has different values (Pre diagnostic IDs only, not Pre + Post)